packages/frontend/src/pages/Profile/Authorize/Authorize.tsx

```diff
@@ -0,0 +1,75 @@
+import { Heading, List, ListItem, type ListItemProps, PageBase, PageNav, Toolbar } from '@altinn/altinn-components';
+import { PlusIcon } from '@navikt/aksel-icons';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { PageRoutes } from '../../routes';
+
+const dummyUsers: ListItemProps[] = [
+  {
+    id: 'party:mathias',
+    title: 'Kristian Haugen',
+    description: 'Eier av Altinn',
+    label: 'Eier',
+    icon: {
+      type: 'person',
+      name: 'Kristian Haugen',
+    },
+  },
+  {
+    id: 'party:bergerbar',
+    title: 'Lillehammer Bakeri',
+    description: 'Bakeri i Lillehammer',
+    icon: {
+      type: 'company',
+      name: 'Lillehammer Bakeri',
+    },
+  },
+];
+
+export const Authorize = () => {
+  const { t } = useTranslation();
+  const [search, setSearch] = useState('');
+
+  const filteredUsers = dummyUsers.filter((user) =>
+    user.title!.toString().toLowerCase().includes(search.toLowerCase()),
```

Comment:
I propose 

```const filteredUsers = dummyUsers.filter((user) =>
  user.title?.toString().toLowerCase().includes(search.toLowerCase()) ?? false,
);```

non-null assertion (!) is just nice to avoid even though it's dummy data and we know it's provided (might forget it later)